google/ggrc-core
# GGRC-5138 Unify Rich Text editor

# Issue description

The application uses two type of editors: wysiwyg (please see wysiwyg.png) and quill (please see quill.png). Due to that there is inconsistency in code and from users perspective editors look differently.

# Steps to test the changes

1. Open an object (for example Control)
2. Move the object into edit mode (for example click on Edit Control)
3. Verify that editor works as expected for Description, Assessment Procedure, Notes fields

# Sanity checklist




# PR Review checklist
